components/calendly_v2/actions/get-event/get-event.mjs

```diff
@@ -1,11 +1,21 @@
-// legacy_hash_id: a_poiwMj
 import { axios } from "@pipedream/platform";
+import { ConfigurationError } from "@pipedream/platform";
+import { URL } from "url";
+
+function getEventUuidFromUrl(event_url) {
+  if (!event_url) {
+    return null;
+  }
+
+  const url = new URL(event_url);
+  return url.pathname.split("/").pop();
+}
 
 export default {
   key: "calendly_v2-get-event",
   name: "Get Event",
-  description: "Gets information about an Event associated with a URI.",
-  version: "0.1.1",
+  description: "Gets information about an Event associated with a URI. [See docs here](https://developer.calendly.com/api-docs/e2f95ebd44914-get-event).",
+  version: "0.1.3",
```

Comment:
@malexanderlim you only need to increment the version when you publish to the registry, so if registry is 0.1.1, yours should be a patch version above (0.1.2).